modules/subnet_router/ecs.tf

```diff
@@ -16,22 +16,24 @@ locals {
   tailscale_volume_name     = "var-lib-tailscale"
   tailscale_definition_path = abspath("${path.module}/container_definitions/tailscale.json")
   tailscale_container_json = templatefile(local.tailscale_definition_path, {
-    hostname           = "${var.vpc}-tailscale"
+    hostname           = local.name
     advertise_routes   = join(",", concat([data.aws_vpc.ecs.cidr_block], var.additional_routes))
+    additional_flags   = var.additional_flags
     auth_key_secret_id = data.aws_secretsmanager_secret.tailscale_auth_key.id
     image_id           = "${data.aws_ecr_repository.tailscale.repository_url}@${data.aws_ecr_image.tailscale.id}"
     volume_name        = local.tailscale_volume_name
     logs_group         = aws_cloudwatch_log_group.tailscale.name
     logs_region        = local.aws_region_name
   })
+  name = var.name != "" ? var.name : "${var.vpc}-tailscale"
```

Comment:
Any reason to use an explicit `""` for the default vs. `null`?

Reply:
Sheer ignorance probably. I'll have a look at how it works with `null`.

Reply:
It still works 👍